app/lib/event_store/repository.rb

```diff
@@ -142,7 +142,6 @@ def initialize(repository = self.class.repository, event_broker = EventBroker.ne
       subscribe_for_notification(:service_deleted, Services::ServiceDeletedEvent)
       subscribe_for_notification(:service_plan_change_requested, Services::ServicePlanChangeRequestedEvent)
 
-      subscribe_event(SegmentSubscriber.new(:account_deleted), Accounts::AccountDeletedEvent)
```

Comment:
We do not send email anymore on account deletion?

Reply:
Right, bug spotted 👍

Reply:
@hallelujah We do, but this line is not related to emails but to segment!